ext/vulnsrc/rhel/rhel.go

```diff
@@ -123,11 +129,17 @@ func (u *updater) Update(datastore database.Datastore) (resp vulnsrc.UpdateRespo
 
 	for _, rhsa := range rhsaList {
 		// Download the RHSA's XML file.
-		r, err := http.Get(ovalURI + rhsaFilePrefix + strconv.Itoa(rhsa) + ".xml")
+		r, err := httputil.GetWithUserAgent(ovalURI + rhsaFilePrefix + strconv.Itoa(rhsa) + ".xml")
 		if err != nil {
 			log.WithError(err).Error("could not download RHEL's update list")
 			return resp, commonerr.ErrCouldNotDownload
 		}
+		defer r.Body.Close()
+
+		if !httputil.Status2xx(r) {
+			log.Errorf("RHEL's update list returned status code %d", r.StatusCode)
```

Comment:
Can you put StatusCode as a field for the log message? Log messages should be constants with their variables stored as fields. This is what lets Clair support JSON logging in addition to the normal format.